qa/TaskImpl.properties.template

```diff
@@ -30,18 +30,9 @@ COMMAND_LINE_1=-classpath
 # file we use command line override of some properties.
 #
 COMMAND_LINE_2=\
-  dist${file.separator}narayana-full-5.13.1.Final-SNAPSHOT${file.separator}etc${file.separator}\
```

Comment:
@jmfinelli is it fine to remove `etc/` folder?

Reply:
Adding the actual jars to the classpath should take care of the `etc/` folder

Reply:
The etc folder from the dist would normally have in it:
https://github.com/jbosstm/narayana/blob/master/narayana-full/src/main/assembly/bin.xml#L195
https://github.com/jbosstm/narayana/blob/master/narayana-full/src/main/assembly/bin.xml#L206
https://github.com/jbosstm/narayana/blob/master/narayana-full/src/main/assembly/bin.xml#L212

Is it fine for qa/ not to have those files?

Reply:
Thanks @jmfinelli - I think the last one is the most interesting as by having the file on the file system it could be readily changed to try different things but thanks again for checking.

Reply:
You're welcome @tomjenkinson :-)

If users want to use `jbossts-properties.xml` for trying things out, they have the option to unzip `narayana-jts-idlj.jar` (command = `unzip narayana-jts-idlj.jar -d narayana-jts-idlj`) and load `jbossts-properties.xml` directly in the classpath. Nevertheless, I think that keeping the file in the jar is the best solution; in fact, as this PR and #2045 prove, QA has not been updated for years

Reply:
I think I was thinking more if a developer wanted to try with different jbossts-properties.xml (this wouldn't show in commit log)

Reply:
I think the above procedure (i.e. extracting `jbossts-properties.xml`) should work also for developers in case they want to try different configurations.

> this wouldn't show in commit log

That `jbossts-properties.xml` should be related to `narayana-jts-idlj.jar`, shouldn't it? If this is the case, developers/users who want to use a different configuration should extract `narayana-jts-idlj.jar` in a folder and add that folder to the classpath.

Reply:
What I was getting at really by `this wouldn't show in commit log` is to respond to `QA has not been updated for years` where if a developer had needed to use the flexibility to add a jbossts-properties.xml to test something in `qa/` locally that would not have shown in changes in `qa/`. The capability (which is being removed) to do that could be considered easier when there is already a folder in this template that would have added a folder to the classpath but I don't suppose it has to be in the template, someone can just manually add it to their own TaskImpl.properties.